akka-persistence-typed/src/main/scala/akka/persistence/typed/PublishedEvent.scala

```diff
@@ -17,10 +18,11 @@ import akka.annotation.DoNotInherit
 trait PublishedEvent {
 
   /** Scala API: When emitted from an Active Active actor this will contain the replica id */
-  def replicaId: Option[String]
+  def replicatedMetaData: Option[ReplicatedPublishedEventMetaData]
```

Comment:
We'll have to make that type non-internal, since this type is public API, but we can do that later

Reply:
👍 yeah i'll make the type public but the field for version vector private